Advance_assignment.php

```diff
@@ -35,44 +35,49 @@
     // var_dump($photo);
 
     $image = 'https://ir-dev-d9.innoraft-sites.com' . $photo->data->attributes->uri->url;
-    // echo "<img src=$image width='' height='' alt='' srcset=''>";
+    // echo "<img src=$image width='' height='' alt='service image' srcset=''>";
     $stitle = ($data['data'][$i]['attributes']['field_secondary_title']['value']);
     // echo $stitle;
     $scontent = ($data['data'][$i]['attributes']['field_services']['value']);
     // echo $scontent;
 
     if ($i % 2 == 0) {
+      echo "<div class='container'>";
       echo "<div class='row flex-row-reverse flex-row gy-5 justify-content-center'>";
-
-      echo "<div class=' col-sm-12 col-md-6  col-xl-4'>";
-
-      echo "<img class='image img-fluid' src='$image'>";
-
+      
+      echo "<div class=' col-sm-12 col-md-6  col-xl-6'>";
+      
+      echo "<img class='image img-fluid' alt='service image' src='$image'>";
+      
       echo "</div>";
-
-      echo "<div class=' col-sm-12 col-md-6  col-xl-4'>";
-
+      
+      echo "<div class=' col-sm-12 col-md-6  col-xl-6'>";
+      
       echo "<div class='text-break'>$stitle</div>";
-
-
+      
+      
```

Comment:
Don't mix PHP and HTML.

Write HTML outside of php tag.

Reply:
> Don't mix PHP and HTML.
> 
> Write HTML outside of php tag.

done
@kunal-sin